buildroot-external/package/licenseinfo/createLicenseHtml.py

```
@@ -0,0 +1,140 @@
+        f.write('<p>This software contains free third party software products used under various license conditions. A list of free software to be used and the license conditions under which the particular free software will be passed on, can be found enclosed below.</p>')
+        f.write('<p>The software is provided “as is” WITHOUT ANY WARRANTY. Our legal liability remains thereby unaffected.</p>\n')
+        f.write('<p>Whenever required by the particular license, we provide you with a complete and machine-readable copy of the free software source text.  Therefor please contact us at support@eq-3.com.</p>\n')
+        f.write('<hr/>\n')
+
+        # write table of contents for packages
+        f.write('<h2>Packages</h2>\n')
+        for entry in manifestEntries:
+            package = entry.get('PACKAGE', 'Unknown Package')
+            f.write(f'<p><a href="#{package}">{package} {entry["VERSION"]}</a></p>\n')
+        f.write('<h2>Licenses</h2>\n')
+        for licName, licHash in commonLicensesDict.items():
```

Ruff (0.15.10)

[warning] Loop control variable `licHash` not used within loop body

Rename unused `licHash` to `_licHash`

(B007)